luxonis_ml/utils/requirements.txt

```diff
@@ -3,6 +3,6 @@ pydantic-settings>=2.1.0
 PyYAML>=6.0
 fsspec>=2023.1.0
 rich>=13.6.0
-s3fs>=2023.1.0
-gcsfs>=2023.1.0
-mlflow>=2.10.0
+# s3fs>=2023.1.0
+# gcsfs>=2023.1.0
+# mlflow>=2.10.0
```

Comment:
Should we remove them entirely from here instead of having them commented?

Reply:
I was thinking it could stay there to document the packages we're installing, but I'm not against removing it altogether.

Reply:
I'll at least add a comment to it